### DISCUSSION TITLE
Permissions files and directories

### DISCUSSION THREAD

**Discussion by ne0xit** [kubernetes-sigs/blob-csi-driver#742] [Q&A]
hello all,

is it possible with the blob-fuse mount to set the permissions of the files and directories separately?
I currently only see the possibility to set only 0770 or 0777 (for files and directories). 

The problem is that any files are executable.

**Comment by andyzhangx** [comment]
I think it's not supported.
here are the details about permission setting in blobfuse mount: https://github.com/Azure/azure-storage-fuse/issues/776#issuecomment-1111022548, since blobfuse is not Unix compatible, the setting is using different way.
